app/src/main/res/layout/fragment_search_time.xml

```diff
@@ -69,5 +70,5 @@
         android:text="@string/pick_date"
         android:textColor="@color/light_grey"
         android:textSize="@dimen/text_size_expanded_title" />
-
 </LinearLayout>
+
```

Comment:
Extra

Reply:
But this new link is automatically added at the and of layout when I commit the file.

Reply:
How to complies play store package in android studio? Please help build is showing error in that package.

Reply:
Choose the playstore flavor and then build the app.

Reply:
Got it. Thanks :smile: